checkST2110.js

```diff
@@ -758,18 +786,31 @@ const test_20_72_4 = (sdp, params) => {
   return errors;
 };
 
-// ST 2110-20 Section 7.2 Test 5 - Check SSN is the required fixed value
+const ssnPermitted20 = ['ST2110-20:2017', 'ST2110-20:2022'];
+
+// ST 2110-20:2017 Section 7.2
+// ST 2110-20:2022 Section 7.2 Test 5 - Check SSN is the required fixed value
 const test_20_72_5 = (sdp, params) => {
   let [mtParams, errors] = extractMTParams(sdp, params);
   for (let stream of mtParams) {
     if (typeof stream.SSN !== 'undefined') {
-      if (stream.SSN !== 'ST2110-20:2017') {
-        errors.push(new Error(`Line ${stream._line}: For stream ${stream._streamNumber}, format parameter 'SSN' is not set to the required value 'ST2110-20:2017', as per ST 2110-20 Section 7.2.`));
+      if (ssnPermitted20.indexOf(stream.SSN) < 0) {
+        errors.push(new Error(`Line ${stream._line}: For stream ${stream._streamNumber}, format parameter 'SSN' is not set to the required value as per ST 2110-20:2017 and ST 2110-20:2022 Section 7.2.`));
+      } else {
+        let detect2022tcs = typeof stream.TCS !== 'undefined' && stream.TCS === 'ST2115LOGS3';
+        let detect2022colorimetry = typeof stream.colorimetry !== 'undefined' && stream.colorimetry === 'ALPHA';
+
+        if ((detect2022tcs || detect2022colorimetry) && stream.SSN !== 'ST2110-20:2022') {
+          errors.push(new Error(`Line ${stream._line}: For stream ${stream._streamNumber}, format parameter 'SSN' is not set to the required value as per ST 2110-20:2022 Section 7.2.`));
+        }
+        if (!(detect2022tcs || detect2022colorimetry) && stream.SSN === 'ST2110-20:2022') {
```

Comment:
Does the spec require that you don't say 2022 in this case?

Reply:
Hmm, maybe it does?

Reply:
Yes, I think so - from the spec:

> SMPTE Standard Number. Senders implementing this standard shall signal the value ST2110-20:2017 unless the colorimetry value ALPHA or the TCS value ST2115LOGS3 are used, in which case the value ST2110-20:2021 shall be signaled.

Reply:
Is that a precise quote? That has `ST2110-20:2021`, not `:2022`?

Reply:
Sorry - the local copy I have is of the previous *uncorrected* version of the spec.  That should be 2022.